systemtests/tests/block-size/testrunner

```diff
@@ -1,53 +1,47 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
What is this block-size test needed for? The block size changes usually only make sense on tape drives. and this test is not run against a tape drive here.

Reply:
Modified to be a tape test.